src/main/java/org/codeforamerica/shiba/output/caf/JobIncomeInformation.java

```diff
@@ -1,13 +1,24 @@
 package org.codeforamerica.shiba.output.caf;
 
+import org.codeforamerica.shiba.Money;
 import org.codeforamerica.shiba.pages.data.Iteration;
 
+import java.math.BigDecimal;
+
 public interface JobIncomeInformation {
     boolean isComplete();
 
-    Double grossMonthlyIncome();
+    Money grossMonthlyIncome();
 
     int getIndexInJobsSubworkflow();
 
     Iteration getIteration();
+
+    default BigDecimal parseWithCommasRemoved(String s) {
+        try {
+            return new BigDecimal(s.replace(",", ""));
```

Comment:
Does this need the `.setScale(0, RoundingMode.DOWN)` ?

Reply:
Removed this in new changes